### ISSUE TITLE
[HW] HIMANSHU YADAV

### ISSUE THREAD

**Issue by Him-Yadav** [datastaxdevs/appdev-week3-graphql#490]
**Name:** HIMANSHU YADAV
**Email:** himanshu9359@gmail.com
**Linkedin Profile:** https://in.linkedin.com/in/himanshu-yadav-43789153

Attach the homework screenshots below:
-----------------------------------------
![image](https://user-images.githubusercontent.com/88819995/129322514-390c5860-13a3-4247-aeff-cc48c66f7543.png)
![image](https://user-images.githubusercontent.com/88819995/129322626-c4af3f43-3639-450a-b384-c1a0fc5bbbaf.png)
![image](https://user-images.githubusercontent.com/88819995/129322819-2a4fd386-11e1-4406-97a9-beb976887dea.png)
![image](https://user-images.githubusercontent.com/88819995/129322916-e7428288-c648-4490-87c1-f649e9b86c28.png)
![image](https://user-images.githubusercontent.com/88819995/129323015-dd0b6673-fd19-480e-8ca3-1d379d49ff84.png)
![image](https://user-images.githubusercontent.com/88819995/129323599-5bd6f669-add0-4722-816c-b5c3d03c95ae.png)
![image](https://user-images.githubusercontent.com/88819995/129323757-1bb3ee52-8696-4522-8b2a-b38933cf23de.png)
![image](https://user-images.githubusercontent.com/88819995/129324025-72ac1bce-609c-461c-adc7-8b6c4cb0abdc.png)
![image](https://user-images.githubusercontent.com/88819995/129324115-9aea00c3-d9ef-48d3-b423-0b44cb37cb43.png)
![image](https://user-images.githubusercontent.com/88819995/129324346-fefa33f9-5802-4bd5-bec6-1f2e2e878675.png)
![image](https://user-images.githubusercontent.com/88819995/129324609-84b54e81-fef9-4120-8fea-32d9894d17ec.png)
![image](https://user-images.githubusercontent.com/88819995/129324680-ecac3796-0927-42d8-90a1-d27978281bad.png)
![image](https://user-images.githubusercontent.com/88819995/129324802-4a2bff34-9679-4406-8d9e-5d063af7f036.png)
![image](https://user-images.githubusercontent.com/88819995/129324941-e5105b3e-de01-4371-99e6-4c58f6a4a3b2.png)
![image](https://user-images.githubusercontent.com/88819995/129325024-1a1ccc86-292d-4263-8041-961ea347552b.png)
![image](https://user-images.githubusercontent.com/88819995/129325124-b3cba586-c55b-423c-a0a2-773d0ae6ffd7.png)

**Comment by SonicDMG** [comment]
Hi Himanshu. Ok, I see you have the application deployed up on Netlify, great job. I'm just missing your custom video. Please add a custom video to your site or point it out please. Thank you.

**Comment by Him-Yadav** [comment]
Hi There, 

Could you please let me know what kind of video I need to make & how to make it?

**Comment by SonicDMG** [comment]
Of course. The video can be anything really and you can simply take the instructions from step 7 https://github.com/datastaxdevs/appdev-week3-graphql#7-insert-values-in-movie-table and modify them to create your own custom video. Maybe name it differently or provide another mp4, totally up to you. The important part is getting how to do the mutation correctly.

**Comment by Him-Yadav** [comment]
Hi,
I have done mutation for movie "Black Rock" , please check below screenshot & provide my Badge, thanks .
![image](https://user-images.githubusercontent.com/88819995/129424289-c78fd0a5-0c00-406f-a283-dbb8fff6e002.png)

**Comment by Him-Yadav** [comment]
Hi,
I have done mutation for movie named "Black Rock", please find screenshot.
If all looks ok now, please provide my Datastax Badge.
![image](https://user-images.githubusercontent.com/88819995/129424525-72bfc5ab-0d9f-4eb3-8503-8ea1650dd660.png)

**Comment by Him-Yadav** [comment]
Please provide an update.

**Comment by HadesArchitect** [comment]
Hey **HIMANSHU YADAV**, great job! Congrats! Here is your badge! https://api.badgr.io/public/assertions/N8ztQB_tRRKLkqMOCiVi-g
